manifests/capacityscheduling/crd.yaml

```diff
@@ -19,7 +19,21 @@ spec:
     singular: elasticquota
   scope: Namespaced
   versions:
-  - name: v1alpha1
+  - additionalPrinterColumns:
+    - description: Used is the current observed total usage of the resource in the
+        namespace.
+      jsonPath: .status.used
+      name: Used
+      type: string
+    - description: Max is the set of desired max limits for each named resource.
+      jsonPath: .spec.max
+      name: Max
+      type: integer
+    - description: Age is the time ElasticQuota was created.
+      jsonPath: .metadata.creationTimestamp
+      name: Age
+      type: date
+    name: v1alpha1
```

Comment:
These fields (3) are not consistent with the fields (6) in config/crd/bases/*

TODO: having 2 copies of the same CRD should be fixed - we should use config/crd/bases/* as the single source of truth, and have all other CRD yaml `ln -s` to config/crd/bases/*

Reply:
There are two crd objects, one pggroup adds 6 fields, and one elasticquota adds 3 fields.

Reply:
Does this todo item need to be fixed in this pr?

Reply:
@Huang-Wei /PTAL

Reply:
> Does this todo need to be fixed in this pr?

We can leave it as a follow-up.